smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/transactions/TransactionalEmitter.java

```diff
@@ -0,0 +1,50 @@
+package io.smallrye.reactive.messaging.kafka.transactions;
+
+import org.eclipse.microprofile.reactive.messaging.Message;
+import org.eclipse.microprofile.reactive.messaging.OnOverflow;
+
+/**
+ * Emitter for sending messages to a Kafka channel in a transaction.
+ * Send methods immediately return by calling the underlying Emitter.
+ *
+ * @param <T> payload type
+ */
+public interface TransactionalEmitter<T> {
+
+    /**
+     * Sends a message to the Kafka channel.
+     * <p>
+     * Immediately returns without waiting the result by dispatches the message to the underlying Emitter.
+     *
+     * @param <M> the <em>Message</em> type
+     * @param msg the <em>Message</em> to send, must not be {@code null}
+     * @throws IllegalStateException if the channel has been cancelled or terminated or if an overflow strategy of
+     *         {@link OnOverflow.Strategy#THROW_EXCEPTION THROW_EXCEPTION} or {@link OnOverflow.Strategy#BUFFER BUFFER} is
+     *         configured and the emitter overflows.
+     * @see io.smallrye.reactive.messaging.MutinyEmitter#send(Message)
+     */
+    <M extends Message<? extends T>> void send(M msg);
+
+    /**
+     * /**
+     * Sends a payload to the channel.
+     * <p>
+     * Immediately returns without waiting the result by dispatches the message to the underlying Emitter.
```

Comment:
```suggestion
     * Immediately returns without waiting for the result by dispatching the message to the underlying Emitter.
```

Is probably what you meant?

Reply:
Exactly :)